cf/data/data.py

```diff
@@ -3804,11 +3805,11 @@ def _binary_operation(self, other, method):
 
         """
         inplace = method[2] == "i"
-        method_type = method[-5:-2]
 
         # ------------------------------------------------------------
         # Ensure that other is an independent Data object
         # ------------------------------------------------------------
+        # TODODASK is this still needed? Comment below was left unfinished.
```

Comment:
I think that is still needed, so that combination with `cf.Query` objects works.

Reply:
Thanks for clarifying in this case as well as those in the other comments, which I will give a :+1: instead of writing the same thing! I'll either commit as suggestions or in new commit(s) to get these resolved.